### PR TITLE
[DUOS-1979][risk=low] Deleted /closed api endpoints and service methods

### PR DESCRIPTION
**Pull request by lu-c**
Am not sure if anything else needs to happen to confirm this was deleted (sorry inexperienced with java)?

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/java/org/broadinstitute/consent/http/resources/ConsentCasesResource.java [hunk 1]
```diff
@@ -81,13 +81,4 @@ public Response getConsentSummaryDetailFile(@QueryParam("type") String type, @Au
...
 
-    @GET
-    @Path("/closed")
-    @Produces("application/json")
-    @RolesAllowed({MEMBER, CHAIRPERSON, ALUMNI, ADMIN})
-    public Response describeClosedElections(@Auth AuthUser authUser) {
-        List<Election> elections = electionService.describeClosedElectionsByType(ElectionType.TRANSLATE_DUL.getValue(), authUser);
-        return Response.ok().entity(elections).build();
-    }
-
```
Comment (rushtong) [inline src/main/java/org/broadinstitute/consent/http/resources/ConsentCasesResource.java:93]: There is an unused import on line 18 due to removing this method

File: src/main/java/org/broadinstitute/consent/http/resources/DataRequestCasesResource.java [hunk 2]
```diff
@@ -60,13 +60,4 @@ public Response getMatchSummaryCases(@Auth AuthUser authUser) {
...
 
-    @GET
-    @Path("/closed")
-    @Produces("application/json")
-    @RolesAllowed({CHAIRPERSON, MEMBER, ALUMNI, ADMIN})
-    public Response describeClosedElections(@Auth AuthUser authUser) {
-        List<Election> elections = electionService.describeClosedElectionsByType(ElectionType.DATA_ACCESS.getValue(), authUser);
-        return Response.ok().entity(elections).build();
-    }
-
```
Comment (rushtong) [inline src/main/java/org/broadinstitute/consent/http/resources/DataRequestCasesResource.java:72]: There are a few unused imports here as a result of this. Also, `electionService` is now unused and we can remove a good chunk of plumbing from that.

**Review by rushtong — COMMENTED** [review]
There's a good deal of clean up from some of this ... I'll submit a PR to this PR with some of that cleanup.

**Comment by rushtong** [comment]
@lu-c If you could take a look at https://github.com/DataBiosphere/consent/pull/1690 and let me know what you think - that should clean up all of the remaining unused code that's fallout from your changes in this PR.

**Review by rushtong — APPROVED** [review]
👍🏽

**Review by connorlbark — APPROVED** [review]
👍

**Review by quazi-h — APPROVED** [review]
Removal of ReviewedCases and api end points looks good to me.

**Pull request merged**